src/Component/Filter/RangeNumericFilter.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Component\Grid\Filter;
+
+use Sylius\Component\Grid\Data\DataSourceInterface;
+use Sylius\Component\Grid\Filtering\FilterInterface;
+
+final class RangeNumericFilter implements FilterInterface
+{
+    public const DEFAULT_SCALE = 0;
+
+    public function apply(DataSourceInterface $dataSource, string $name, $data, array $options): void
+    {
+        if (empty($data)) {
+            return;
+        }
+
+        $field = $options['field'] ?? $name;
```

Comment:
I assume if I will write `field: 1`, then it would be int and we need string. `Date` filter do this conversion as well.
```suggestion
        $field = (string) $options['field'] ?? $name;
```